ssl/src/main/java/io/crate/protocols/ssl/SslConfigSettings.java

```diff
@@ -76,7 +77,9 @@ private SslConfigSettings() {}
     public static final CrateSetting<TimeValue> SSL_RESOURCE_POLL_INTERVAL = CrateSetting.of(
         Setting.timeSetting(
             SSL_RESOURCE_POLL_INTERVAL_NAME,
-            TimeValue.timeValueSeconds(5),
+            TimeValue.timeValueSeconds(SensitivityWatchEventModifier.MEDIUM.sensitivityValueInSeconds()), // default
+            TimeValue.timeValueSeconds(SensitivityWatchEventModifier.HIGH.sensitivityValueInSeconds()),
```

Comment:
I'm not sure if adding a strict min/max value as part of a fix isn't too restrictive. How about we add a validator that logs a warning if an invalid value is used, but still accept it ?

Reply:
Right, this may to much for a hotfix. Would you also not merge these restriction to master and use the logging validator there as well? I'd be fine for me as well.

Reply:
For master/4.1 it would be fine I guess (with a breaking change entry in the release notes then?).
Maybe make first the commit for the fix, and follow up with the change to make it more restrictive?